internal/resource/sysfs-device.go

```diff
@@ -68,3 +68,11 @@ func (d vfioDevice) IsMigCapable() (bool, error) {
 func (d vfioDevice) GetPCIClass() (uint32, error) {
 	return d.nvidiaPCIDevice.Class, nil
 }
+
+func (d vfioDevice) GetClusterUUID() (string, error) {
+	return "", nil
```

Comment:
Isn't having a nil value here going to cause the loop in `newImexDomainLabeler()` to error out when comparing against `commonClusterUUID`?

Reply:
addressed